### ISSUE TITLE
Disable plugins autoupdate.

### ISSUE THREAD

**Issue by Zatoichy** [supermarin/Alcatraz#236]
Please add the configuration option to disable auto-update plug-ins.

**Comment by supermarin** [comment]
@Zatoichy what's the usecase for that?

**Comment by Zatoichy** [comment]
Update (improvement) of some plug-ins led to a disruption of xcode.

**Comment by supermarin** [comment]
@Zatoichy would you mind posting which plugins caused the issue?

**Comment by Zatoichy** [comment]
For example: MCLog. https://github.com/yuhua-chen/MCLog/issues/9

**Comment by supermarin** [comment]
Hmm.. I'm not sure how to feel about that.
I think we should shoot for more automation and somehow have a way of testing if Xcode crashes with the new plugins when they're added / updated.

Putting the plugin updates to the users manually seems like a degradation of the service.
@kattrali @jurre thougths?

**Comment by Zatoichy** [comment]
I do not call to remove auto-update. A choice to not be redundant.

**Comment by jurre** [comment]
Yeah I'm not a fan of disabling auto-update altogether (even by user choice).

Maybe we could let users specify a git tag/commit for specific plugins?  Won't work with binary installs though, in that case they might specify a release nr?

**Comment by kattrali** [comment]
> Maybe we could let users specify a git tag/commit for specific plugins? Won't work with binary installs though, in that case they might specify a release nr?

This seems plausible for the `alcatrazrc` file, if people prefer to lock their plugins to a particular version. Though I'd say it seems like something uncommon enough that it doesn't need new UI, especially since locking versions might hide a different set of unintended behaviors, like plugins going missing between xcode versions because they never updated to the latest version with new UUIDs.

**Comment by supermarin** [comment]
Yeah makes sense. I think automation is a better answer here,
and if a certain plugin keeps breaking, we should remove it from the list.
